l10n_lu_mis_reports/data/mis_report_abr_pl.xml

```diff
@@ -0,0 +1,393 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+    <record model="mis.report" id="mis_report_abr_pl">
+      <field name="name">Luxembourg Profit &amp; Loss (abbreviated)</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_abr_pl_fr429">
+      <field name="report_id" ref="mis_report_abr_pl"/>
+      <field name="name">fr429</field>
+      <field name="description">A. CHARGES</field>
+      <field name="expression">fr430 + fr431 + fr436 + fr439 + fr440 + fr441 + fr442 + fr445 + fr446 + fr447 + fr464</field>
+      <field name="suffix">€</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="default_css_style">background-color: orange; font-weight: bold</field>
+      <field name="sequence">10</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_abr_pl_fr430">
+      <field name="report_id" ref="mis_report_abr_pl"/>
+      <field name="name">fr430</field>
+      <field name="description">1. à 2. Charges brutes (moins B.1 à B.4)</field>
+      <field name="expression">max(balp[60%,61%,70%,71%,72%,73%], 0)</field>
```

Comment:
I get the following error :

title="fr430 = max(balp[60%,61%,70%,71%,72%,74%], 0)

Traceback (most recent call last):
  File "/home/lga/project/odoo/addons-account-financial-reporting/mis_builder/models/mis_builder.py", line 527, in _compute
    kpi_val = safe_eval(kpi_eval_expression, localdict)
  File "/home/lga/project/odoo/parts/odoo/openerp/tools/safe_eval.py", line 314, in safe_eval
    return eval(c, globals_dict, locals_dict)
  File "", line 1, in <module>
ValueError: "_max() takes exactly 1 argument (2 given)" while evaluating
u'max((0.0), 0)'
"